linux/systemd/qubes-core.service

```diff
@@ -10,10 +10,11 @@ RemainAfterExit=yes
 # Needed to avoid rebooting before all VMs have shut down.
 TimeoutStopSec=180
 ExecStart=/usr/lib/qubes/startup-misc.sh
+ExecStop=-/usr/bin/pkill qubes-guid
```

Comment:
`ExecStop=-/usr/bin/pkill qubes-guid`
Just a reminder that this line should no longer be necessary, because [this PR](https://github.com/QubesOS/qubes-gui-daemon/pull/24) ensures that `qubes-guid` is no longer present at this point in time (there [was](https://github.com/QubesOS/qubes-issues/issues/1581#issuecomment-420847298) only one process but it required SIGKILL to terminate anyway and it was required at an [earlier](https://github.com/QubesOS/qubes-issues/issues/1581#issuecomment-420392892) point, eg. after `graphical.target` stops)